contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -250,7 +250,7 @@ func recoverAssignedSlave(pod *api.Pod) string {
 
```

Comment:
This [commit](https://github.com/kubernetes/kubernetes/commit/2c4142494a4e2d61f36814341a38f9b1a69745de) changed "plugin.go" to "podreconciler.go".

It would require a rebase.